swim/handlers.go

```diff
@@ -155,7 +155,10 @@ func (n *Node) adminLeaveHandler(ctx json.Context, req *emptyArg) (*Status, erro
 }
 
 func (n *Node) adminSetLabelHandler(ctx json.Context, req *SetLabelRequest) (*Status, error) {
-	n.Labels().Set(req.Key, req.Value)
+	err := n.Labels().Set(req.Key, req.Value)
+	if err != nil {
+		return nil, err
+	}
 	return &Status{Status: "ok"}, nil
```

Comment:
`Set` returns false when nothing changed right? Maybe nice to put that in the response as well?